docs/components/checkbox-field/index.md

```diff
@@ -103,73 +103,142 @@ Target the error block via `data-error`.
 
 ## All UI States
 
-<div class="flex flex-col space-y-4" style="max-width: 14rem">
-<Form::CheckboxField
-@label='Label'
-/>
+### CheckboxField with label
 
-<Form::CheckboxField
-@label='Label'
-@error="With error"
-/>
+<div class='mb-4 w-64'>
+  <Form::CheckboxField
+    @label='This is an option that expands to multiple lines'
+  />
+</div>
 
-<Form::CheckboxField
-@label='Label'
-@hint='With hint text'
-/>
+### CheckboxField with label and hint
```

Comment:
Love how this looks! Will try to update the File Input Field with these headers as well (I'm on two related FileInput PRs, so maybe can put it in one of those)

Reply:
See: https://github.com/CrowdStrike/ember-toucan-core/pull/110/commits/72b72da48c4c4cd88f9ae7e855376a69f249511b